try {
    doesntExist();
}
catch (err){
   

    let onlyHere = true;
    let outerVariable = true;
}

console.log(outerVariable);
